### PR TITLE
SEO-174801 - JS-Multiple-H1

### PR DESCRIPTION
**Pull request by amyprudencekingstone**
Hi @MallikaRK ,
Worked , alt text correction, image name correction,d and description. Kindly review.

Regards,
Amy Kingstone.

### CODE REVIEW

File: js/AureliaJS.md [hunk 1]
```diff
@@ -249,9 +249,9 @@ gulp watch
 
 {% endhighlight %}
 
-* Browse to [`http://localhost:9000`](http://localhost:9000)  to see the application. You can make changes in the code found under `src` folder and the browser should auto-refresh itself while you save files. The Grid component is rendered as like the below screenshot.
+* Browse to [`http://localhost:9000`](http://localhost:9000/)  to see the application. You can make changes in the code found under `src` folder and the browser should auto-refresh itself while you save files. The Grid component is rendered as like the below screenshot.
 
-![](/js/Aurelia_images/skeleton_navigation.png)
+![Skeleton navigation in JavaScipt Aurelia](/js/aurelia_images/javascript-aurelia-skeleton-navigation.png)
 
 # Catalog application
```
Comment (MallikaRK) [inline js/AureliaJS.md:256]: why # H1 tag is added here?

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: js/AureliaJS.md (hunk 1 above)
Comment (amyprudencekingstone) [reply js/AureliaJS.md]: made changes

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**